src/linux/Packaging.Linux/install-from-source.sh

```diff
@@ -225,5 +240,5 @@ if [ -z "$DOTNET_ROOT" ]; then
 fi
 
 cd "$toplevel_path"
-$sudo_cmd env "PATH=$PATH" $DOTNET_ROOT/dotnet build ./src/linux/Packaging.Linux/Packaging.Linux.csproj -c Release -p:InstallFromSource=true
-add_to_PATH "/usr/local/bin"
+$sudo_cmd env "PATH=$PATH" $DOTNET_ROOT/dotnet build ./src/linux/Packaging.Linux/Packaging.Linux.csproj -c Release -p:InstallFromSource=true -p:InstallLocation=$installLocation
+add_to_PATH $installLocation
```

Comment:
`$installLocation` will be something like `/usr/local` but we want to add `/usr/local/bin` to the `$PATH`.. we should therefore be using something like this instead here:

```shell
add_to_PATH "${installLocation}/bin"
```

Reply:
Good catch! I also agree with the "prefix" name change.

Reply:
@m4ss1m0g - I opened https://github.com/m4ss1m0g/git-credential-manager/pull/2 to address these comments. Once merged, I think we will _actually_ be good to go here 😀.

Reply:
Thanks